.buildkite/pipeline.yml

```diff
@@ -0,0 +1,32 @@
+steps:
```

Comment:
Why is Buildkite being used instead of GH actions?

Reply:
Buildkite is preferred since aws keys can be stored more securely over admiral secrets rather than storing them as github secrets on a public repository.